modules/installation-special-config-custom-image.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-special-config-custom-image_{context}"]
+= Generating and applying a custom override base image
+
+In {product-title} {product-version}, the {op-system-first} component
+is now shipped as a standard container image, and it is supported to
+create customized versions using standard container build tooling.
+
+To find the container image associated with a specific {product-title}
+version, use `oc adm release info --image-for rhel-coreos-8 <image>`
+on the release image reported by `oc get clusterversion`.
+
+== Generating a container image with overrides
+
+This example `Dockerfile` demonstrates using `rpm-ostree override replace`
+to generate a container image that replaces the `NetworkManager` package:
+
+```
+FROM <operating system base>
```

Comment:
what should be written as operating-system-base? maybe add a comment with an example?

Reply:
This is the section above on finding the image, which is annoying and painful until we fix https://issues.redhat.com/browse/ART-4352